mantidproject/fitbenchmarking
# Added More Tests

#### Description of Work

Fixes #142 

Added more tests and got rid of redundant ones.
Will add the test suite for the scipy part of the code in a further pull request.

#### Testing Instructions

Check if the automatic tests pass. Additionally, you could check what passes and how by going to the travis page.
